virkailija/src/routes/Yllapito.tsx

```diff
@@ -50,6 +50,10 @@ const Header = styled(Heading)`
   padding-right: 10px;
 `
 
+const HakuInput = styled("input")`
+  width: 11rem;
```

Comment:
Fixed sizes are not responsive. If really needed use `width: 100%; max-size: 11rem;` here (or with display: block or flex positioning)